### PR TITLE
test(Rating): support new a11y rule of radio groups

### PR DESCRIPTION
**Pull request by mark-ck**
https://a11y-style-guide.com/style-guide/section-forms.html#kssref-forms-radio-buttons

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
         "@no-repeat/sassdoc-parser": "^0.0.11",
         "@octokit/rest": "^15.15.1",
         "autoprefixer": "^7.1.4",
-        "axe-core": "3.1.2",
+        "axe-core": "^3.1.2",
```
Comment (jdkahn) [inline package.json]: Change this to 3.2.0 and then run npm install and test

File: src/rating/rating.jsx [hunk 2]
```diff
@@ -431,7 +431,16 @@ export default class Rating extends Component {
                         className={`${prefix}rating-overlay`}
                         style={overlayStyle}
                     >
-                        <form action="#">{overlay}</form>
+                        <form action="#">
+                            {enableA11y ? (
+                                <div role="group" aria-labelledby={id}>
```
Comment (jdkahn) [inline src/rating/rating.jsx]: the div above with this `id` needs to have an `aria-label`

File: package.json [hunk 3]
```diff
@@ -85,7 +85,7 @@
         "@no-repeat/sassdoc-parser": "^0.0.11",
         "@octokit/rest": "^15.15.1",
         "autoprefixer": "^7.1.4",
-        "axe-core": "3.1.2",
+        "axe-core": "3.2.0",
```
Comment (mark-ck) [inline package.json]: @jdkahn i have change this to `3.2.0`, it also passed the check.

**Comment by youluna** [comment]
@jdkahn  still have requested changes? If not you can merge this pr~

File: src/rating/rating.jsx [hunk 4]
```diff
@@ -433,9 +433,12 @@ export default class Rating extends Component {
                     >
                         <form action="#">
                             {enableA11y ? (
-                                <div role="group" aria-labelledby={id}>
-                                    {' '}
-                                    overlay{' '}
+                                <div
+                                    role="group"
+                                    aria-labelledby={id}
+                                    aria-label="Rating Options"
```
Comment (jdkahn) [inline src/rating/rating.jsx]: @mark-ck the `aria-label` should be attached to wherever the `aria-labelledby` points to. And "Rating Options` is in english so we should add i18n support

Comment (mark-ck) [reply src/rating/rating.jsx]: I have noticed that😂.
I tried that the `aria-label` won't be read by `VoiceOver`, so i didn't add i18n support.
But you are right, i'll add i18n support later.

**Comment by jdkahn** [comment]
I made the changes below and it passed axe-core 3.2 and VoiceOver read easily with no repeated phrases. 
screen shot 2019-03-07 at 10 22 36 am
screen shot 2019-03-07 at 10 22 24 am

File: package.json (hunk 1 above)
Comment (jdkahn) [reply package.json]: This needs to be changed to 3.2 because we need developers to install the latest

File: package.json [hunk 5]
```diff
@@ -85,7 +85,7 @@
     "@no-repeat/sassdoc-parser": "^0.0.11",
     "@octokit/rest": "^15.15.1",
     "autoprefixer": "^7.1.4",
-    "axe-core": "3.1.2",
+    "axe-core": "^3.1.2",
```
Comment (jdkahn) [inline package.json]: needs to be updated to 3.2

File: src/rating/rating.jsx [hunk 6]
```diff
@@ -419,6 +426,8 @@ export default class Rating extends Component {
                 className={ratingCls}
                 onKeyDown={this.onKeyDown}
                 tabIndex="0"
+                role="group"
+                aria-label={locale.description}
```
Comment (jdkahn) [inline src/rating/rating.jsx]: - remove `aria-labelledby={id}` on line 365
- add `aria-hidden` to underlay div on line 435

Comment (mark-ck) [reply src/rating/rating.jsx]: done

**Pull request merged**